tunneling/fastify/index.js

```diff
@@ -0,0 +1,49 @@
+const Fastify = require("fastify");
+
+const fastify = Fastify({ logger: true, bodyLimit: 100 * 1048576 }); //TODO: adjust bodyLimit to suit your needs (Default value is 100mb to match cloudflare free plan limits)
+
+
+//TODO: Change this to suit your needs
+const SENTRY_HOST = "oXXXXXX.ingest.sentry.io";
+const SENTRY_KNOWN_PROJECTS = ["123456"]
+
+//TODO: Put your routes here, make sure to keep the /bugs route at the end of your file
+
+
+//Handle replay packets where no content-type header is available
+fastify.addContentTypeParser("*", { parseAs: "buffer" }, function (req, body, done) {
+    done(null, body);
+});
+fastify.post("/bugtracker", async (request, reply) => {

```

Comment:
```suggestion
fastify.post("/bugs", async (request, reply) => {
```

to keep this consistent

Reply:
I forgot sorry